src/mrnet/core/reactions.py

```diff
@@ -1621,11 +1703,43 @@ def graph_representation(self,) -> nx.DiGraph:  # temp here, use graph_rep_1_2 i
         elif len(self.reactants) == 1 and len(self.products) == 2:
             return graph_rep_1_2(self)
         elif len(self.reactants) == 2 and len(self.products) == 1:
-            self.reactants, self.products = self.products, self.reactants
+            self.swap_elements()
             return graph_rep_1_2(self)
         elif len(self.reactants) == len(self.products) == 2:
             return graph_rep_2_2(self)
 
+    def swap_elements(self):
+        self.reactants, self.products = self.products, self.reactants
+        self.base_free_energy_A, self.base_free_energy_B = (
+            self.base_free_energy_B,
+            self.base_free_energy_A,
+        )
+        self.reactant_energy, self.product_energy = (
+            self.product_energy,
+            self.reactant_energy,
+        )
+        self.reactant_enthalpy, self.product_enthalpy = (
+            self.product_enthalpy,
+            self.reactant_enthalpy,
+        )
+        self.reactant_entropy, self.product_entropy = (
+            self.product_entropy,
+            self.reactant_entropy,
+        )
+        self.reactant_charge, self.product_charge = (
+            self.product_charge,
+            self.reactant_charge,
+        )
+        self.reactant_eids, self.product_eids = self.product_eids, self.reactant_eids
+        self.reactant_indices, self.product_indices = (
+            self.product_indices,
+            self.reactant_indices,
+        )
+        self.reactant_atom_mapping, self.product_atom_mapping = (
+            self.product_atom_mapping,
+            self.reactant_atom_mapping,
+        )
+
```

Comment:
So this is needed because, as of right now if we have a Concerted Reaction with 2 reactants and 1 product, we essentially compute the graph representation by flipping the direction of the reaction. Perhaps this will truly become vestigial once the extra nodes are removed from the network.